crates/noirc_evaluator/src/ssa_refactor/ir/instruction.rs

```diff
@@ -150,14 +150,9 @@ pub(crate) enum TerminatorInstruction {
     ///
     /// Jump If
     ///
-    /// If the condition is true: jump to the specified `then_destination` with `arguments`.
-    /// Otherwise, jump to the specified `else_destination` with `arguments`.
-    JmpIf {
-        condition: ValueId,
-        then_destination: BasicBlockId,
-        else_destination: BasicBlockId,
-        arguments: Vec<ValueId>,
```

Comment:
I removed it because we never need it for our ssa gen pass. `if` and `for` are the only constructs that conditionally jump and neither need the extra arguments. I suppose there are alternate ways of encoding `for` that would use it but I did not end up encoding `for` this way in the for PR.